salt/elife-xpub/init.sls

```diff
@@ -159,6 +159,27 @@ elife-xpub-nginx-vhost:
         - listen_in:
             - service: nginx-server-service
 
+# makes Grobid eagerly load models
+# https://github.com/kermitt2/grobid/issues/289
+sciencebeam-grobid-models-warmup:
+    file.managed:
+        - name: /home/{{ pillar.elife.deploy_user.name }}/warmup.pdf
+        - source: salt://elife-xpub/config/home-deploy-user-warmup.pdf
+
+    cmd.run:
+        - name: |
+            curl -X POST \
+            --fail \
+            --show-error \
+            -H "Content-Type: application/pdf" \
+            --data-binary @warmup.pdf \
```

Comment:
where is `warmup.pdf` coming from?

Reply:
Added comment with source URL, if there's another one it can be changed